### ISSUE TITLE
Show "Changing Shield settings in this way..." only once in Private Window

### ISSUE THREAD

**Issue by 360fun** [brave/brave-browser#12156]
I find really annoying to see "Changing Shield settings in this way may affect web compatibility on this site." **every time I try to disable the Ads Shield on a new Private Window**.

![image](https://user-images.githubusercontent.com/844090/96122153-60456580-0ef1-11eb-856b-ddfed2cfa2f8.png)

I would like Brave to **show it just once** and, after pressing the OK button, **never show it again** (this is the behaviour I expected).

**Comment by karenkliu** [comment]
@rebron Correct. Let's not show this message for Private & Private Window with Tor.
